ui/src/app/user/user.component.ts

```diff
@@ -97,88 +93,81 @@ export class UserComponent implements OnInit {
 
   public enableAndDisableFormFields(): boolean {
     // Update Fields
-    this.form?.fields[0].fieldGroup.forEach(element => {
+    this.userInformationFields.forEach(element => {
       element.templateOptions.disabled = !element.templateOptions.disabled;
     });
     return this.isEditModeDisabled = !this.isEditModeDisabled;
   }
 
-
-  public getFields(): FormlyFieldConfig[] {
-
-    return [{
-      fieldGroup: [
-        {
-          key: "firstname",
-          type: "input",
-          templateOptions: {
-            label: this.translate.instant("Register.Form.firstname"),
-            disabled: true
-          }
-        },
-        {
-          key: "lastname",
-          type: "input",
-          templateOptions: {
-            label: this.translate.instant("Register.Form.lastname"),
-            disabled: true
-          }
-        },
-        {
-          key: "street",
-          type: "input",
-          templateOptions: {
-            label: this.translate.instant("Register.Form.street"),
-            disabled: true
-          }
-        },
-        {
-          key: "zip",
-          type: "input",
-          templateOptions: {
-            label: this.translate.instant("Register.Form.zip"),
-            disabled: true
-          }
-        },
-        {
-          key: "city",
-          type: "input",
-          templateOptions: {
-            label: this.translate.instant("Register.Form.city"),
-            disabled: true
-          }
-        },
-        {
-          key: "country",
-          type: "select",
-          templateOptions: {
-            label: this.translate.instant("Register.Form.country"),
-            options: COUNTRY_OPTIONS(this.translate),
-            disabled: true
-          }
-        },
-        {
-          key: "email",
-          type: "input",
-          templateOptions: {
-            label: this.translate.instant("Register.Form.email"),
-            disabled: true
-          },
-          validators: {
-            validation: [Validators.email]
-          }
-        },
-        {
-          key: "phone",
-          type: "input",
-          templateOptions: {
-            label: this.translate.instant("Register.Form.phone"),
-            disabled: true
-          }
-        }
-      ]
-    }];
-  }
+  /** Needs to be predefined to make wrapper work with ion-skeleton */
+  protected userInformationFields = [{
```

Comment:
last requested change, can you add the corresponding type here?

Reply:
can you also refactor `templateOptions` to `props` since `templateOptions` are deprecated